src/services/envVersion/envVersion.test.tsx

```diff
@@ -43,7 +43,12 @@ const mockGetEnvVersionValidResponse = {
   features: {
     REQUIRE_OTP: true,
     TRANSACTION_GROUPING: true,
-    FLOW_TYPE: "DEFAULT"
+    FLOW_TYPE: "DEFAULT",
```

Comment:
Hmm, is there a reason why we initialise this multiple times ah. Rather than centralising it

Reply:
hmm this is for the test case, used to mockFetch the /version endpoint!

Reply:
I see I see! okok haha